lib/streams/copy-stream.js

```diff
@@ -64,4 +66,18 @@ module.exports = class CopyStream extends AbstractWritableStream {
             .then(() => callback())
             .catch(callback);
     }
+    /**
+     * Adds symlink to artifact.
+     *
+     * @param {{path: string, relative: string, base: string, cwd: string, stats: fs.Stats}} file — the file info.
+     * @param {function} callback — call this function when processing is complete.
+     * @abstract
+     */
+    addSymbolicLink(file, callback) {
+        const dest = path.join(this._dest, file.relative);
+
+        copySymlink(file.path, dest, { fs })
+            .then(() => callback())
+            .catch(callback);
```

Comment:
Why not `() => callback()` as a line above?

Reply:
because `.catch(err => callback(err))`

Reply:
👌